### ISSUE TITLE
[tool_crash] ClientException

### ISSUE THREAD

**Issue by Zasaz** [flutter/flutter#90587]
## Command
```
flutter --no-color run --machine --track-widget-creation --device-id=emulator-5554 --start-paused --dart-define=flutter.inspector.structuredErrors=true --devtools-server-address=http://127.0.0.1:9102 lib\main.dart
```

## Steps to Reproduce
1. I am working on maps and when i try to test it then this error pops up.

## Logs
ClientException
```
#0      IOClient.send.<anonymous closure> (package:http/src/io_client.dart:49:13)
#1      _invokeErrorHandler (dart:async/async_error.dart:45:24)
#2      _HandleErrorStream._handleError (dart:async/stream_pipe.dart:269:9)
#3      _ForwardingStreamSubscription._handleError (dart:async/stream_pipe.dart:157:13)
#4      _HttpClientResponse.listen.<anonymous closure> (dart:_http/http_impl.dart:709:16)
#5      _rootRunBinary (dart:async/zone.dart:1452:47)
#6      _CustomZone.runBinary (dart:async/zone.dart:1342:19)
#7      _CustomZone.runBinaryGuarded (dart:async/zone.dart:1252:7)
#8      _BufferingStreamSubscription._sendError.sendError (dart:async/stream_impl.dart:360:15)
#9      _BufferingStreamSubscription._sendError (dart:async/stream_impl.dart:378:7)
#10     _BufferingStreamSubscription._addError (dart:async/stream_impl.dart:280:7)
#11     _ForwardingStreamSubscription._addError (dart:async/stream_pipe.dart:128:11)
#12     _addErrorWithReplacement (dart:async/stream_pipe.dart:176:8)
#13     _HandleErrorStream._handleError (dart:async/stream_pipe.dart:274:11)
#14     _ForwardingStreamSubscription._handleError (dart:async/stream_pipe.dart:157:13)
#15     _rootRunBinary (dart:async/zone.dart:1452:47)
#16     _CustomZone.runBinary (dart:async/zone.dart:1342:19)
#17     _CustomZone.runBinaryGuarded (dart:async/zone.dart:1252:7)
#18     _BufferingStreamSubscription._sendError.sendError (dart:async/stream_impl.dart:360:15)
#19     _BufferingStreamSubscription._sendError (dart:async/stream_impl.dart:378:7)
#20     _BufferingStreamSubscription._addError (dart:async/stream_impl.dart:280:7)
#21     _SyncStreamControllerDispatch._sendError (dart:async/stream_controller.dart:737:19)
#22     _StreamController._addError (dart:async/stream_controller.dart:615:7)
#23     _StreamController.addError (dart:async/stream_controller.dart:569:5)
#24     _HttpParser._reportBodyError (dart:_http/http_parser.dart:1183:22)
```
```
[✓] Flutter (Channel unknown, 2.5.0, on Microsoft Windows [Version 10.0.19043.1237], locale en-US)
    • Flutter version 2.5.0 at C:\src\flutter
    • Upstream repository unknown
    • Framework revision 4cc385b4b8 (2 weeks ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:/Users/Umair Ahmad/AppData/Local/Android/Sdk
    ✗ cmdline-tools component is missing
      Run `path/to/sdkmanager --install "cmdline-tools;latest"`
      See https://developer.android.com/studio/command-line for more details.
    ✗ Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[✗] Chrome - develop for the web (Cannot find Chrome executable at .\Google\Chrome\Application\chrome.exe)
    ! Cannot find Chrome. Try setting CHROME_EXECUTABLE to a Chrome executable.

[✓] Android Studio (version 4.2)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[✓] VS Code (version 1.60.2)
    • VS Code at C:\Users\Umair Ahmad\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.26.0

[✓] Connected device (2 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86    • Android 8.1.0 (API 27) (emulator)
    • Edge (web)                         • edge          • web-javascript • Microsoft Edge 93.0.961.52

! Doctor found issues in 2 categories.

```

## Flutter Application Metadata
**Type**: app
**Version**: 1.0.0+1
**Material**: true
**Android X**: false
**Module**: false
**Plugin**: false
**Android package**: null
**iOS bundle identifier**: null
**Creation channel**: beta
**Creation framework version**: f18b9281c2280c2646aa3d4348715ed5bb9446c8
### Plugins
flutter_plugin_android_lifecycle-2.0.3
google_maps_flutter-2.0.9
location-4.3.0
location_web-3.1.1
path_provider-2.0.3
path_provider_linux-2.1.0
path_provider_macos-2.0.2
path_provider_windows-2.0.3

**Comment by darshankawar** [comment]
@Zasaz 
This issue was fixed and merged into master, #73356
Please switch to latest master and try the same command again to see if it works or issue still persists.

Closing this for now as this is already fixed. If after switching to latest master you see the same issue again, write in comments and I'll reopen it.

**Comment by Zasaz** [comment]
Thank you issue is fixed
